cms/blocks.py

```diff
@@ -10,7 +10,7 @@ class CourseTeamBlock(blocks.StructBlock):
     """
 
     name = blocks.CharBlock(max_length=100, help_text="Name of the course team member.")
-    title = blocks.CharBlock(max_length=200, help_text="Title of the course team member.")
+    title = blocks.RichTextBlock(blank=True, null=True, help_text="Title of the course team member.")
```

Comment:
I believe you should disable the features not needed in this place, specifically link, document, image, lists etc. We can stay with the bold and italic if needed. Personally I think even h2, h3 etc need to be removed from this field because they have the potential to break the UI.

Reply:
updated